_Analyse Neurons.ijm

```diff
@@ -232,7 +232,11 @@ file_name_length=lengthOf(file_name_full);
 if(file_name_length>50)
 {
 	file_name=substring(file_name_full, 0, 20); //Restricting file name length as in Windows long path names can cause errors
-	suffix = getString("File name is too long, so it will be truncated. Enter custom name to be be added to end of filename", "_1");
+	Dialog.create("The file name is too long, instead write a Custom Identifier for this Image");
```

Comment:
Is this dialog box to display more  info in the `addMessage `box?

Reply:
Yes, instead of a single line of text I added a dialogue box with a message and example of how this "custom identifer" modifies the image name